rust/syntect-plugin/Makefile

```diff
@@ -10,7 +10,8 @@ out/syntect: $(lang_configs) xi-syntect-plugin
 	mkdir -p out/syntect/bin
 	cp $(lang_configs) out/syntect
 	cp ../target/release/xi-syntect-plugin out/syntect/bin
-	cp manifest.toml out/syntect/manifest.toml
+	cp manifest.toml out/syntect/
+	cp syntaxes.packfile out/syntect/
```

Comment:
I think actually making this packfile should be a target in the makefile? There's an example of what this might look like in [the syntect repo](https://github.com/trishume/syntect/blob/master/Makefile)